packages/gatsby/src/redux/actions/public.ts

```diff
@@ -505,7 +496,11 @@ ${reservedFields.map(f => `  * "${f}"`).join(`\n`)}
  * @example
  * deleteNode({node: node})
```

Comment:
we want to remove the `params/returns/example/etc` from the js docs, mind doing that?

Reply:
Done in 98484a9